spec/spec_helper.rb

```diff
@@ -11,4 +11,14 @@
       social_share_button.allow_sites = %w(twitter facebook whatsapp_app whatsapp_web telegram)
     end
   end
+
+  Decidim::Verifications.register_workflow(:another_dummy_authorization_handler) do |workflow|
+    workflow.form = "AnotherDummyAuthorizationHandler"
+    workflow.action_authorizer = "DummyAuthorizationHandler::DummyActionAuthorizer"
+  end
+
+  Decidim::Verifications.register_workflow(:dummy_authorization_handler) do |workflow|
+    workflow.form = "DummyAuthorizationHandler"
+    workflow.action_authorizer = "DummyAuthorizationHandler::DummyActionAuthorizer"
+  end
```

Comment:
Thanks for the contribution, but the config must stay in the initializer and not in the spec_helper IMHO